packages/bsky/proto/bsky.proto

```diff
@@ -487,6 +495,7 @@ message ActorInfo {
   Record status_record = 17;
   string allow_activity_subscriptions_from = 18;
   optional AgeAssuranceStatus age_assurance_status = 19;
+  Record germ_record = 20;
```

Comment:
how big are the continuity proofs in the record? & do we want to be passing that whole thing from the dataplane to the application if it doesn't go in the view?